curvefs/src/client/s3/client_s3.cpp

```diff
@@ -63,6 +63,11 @@ int S3ClientImpl::Download(const std::string &name, char *buf, uint64_t offset,
             << ",length:" << length;
     ret = s3Adapter_->GetObject(name, buf, offset, length);
     if (ret < 0) {
+        const Aws::String aws_key(name.c_str(), name.size());
```

Comment:
Why not use a separate interface, but put it in getobject

Reply:
won't fix. s3 sdk has been wrapped so many times...